frontend/src/learning/components/ItemDisplay/WordDisplay/PopUp.js

```diff
@@ -0,0 +1,89 @@
+
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components';
+import RadicalDisplay from '../RadicalDisplay/RadDisplay';
+import WordDisplay from './WordDisplay';
+import PropTypes from 'prop-types';
+
+//Can't render if uncomment the following line. Why?
+//Modal.setAppElement('#root');
```

Comment:
Because we don't have an element with `id=root`. As you can see in `frontend/index.js`, the root element id is `learning-app`.